src/main/java/uk/gov/hmcts/ccd/domain/model/definition/CaseField.java

```diff
@@ -123,4 +125,8 @@ public boolean isMetadata() {
     public void setMetadata(boolean metadata) {
         this.metadata = metadata;
     }
+
+    public boolean collectionFieldType() {
```

Comment:
shall we rename to `isCollectionFieldType`?

Reply:
done